test/e2e/helper.go

```diff
@@ -21,6 +21,7 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/onsi/ginkgo/v2"
```

Comment:
I see that we use ginkgo v1 in the other files. Can we update them all? If it is a lot of work, then we can use v1 here as well to keep it consistent across the files.